yarn.lock

```diff
@@ -982,11 +982,25 @@ clean-stack@^2.0.0:
   resolved "https://registry.yarnpkg.com/clean-stack/-/clean-stack-2.2.0.tgz#ee8472dbb129e727b31e8a10a427dee9dfe4008b"
   integrity sha512-4diC9HaTE+KRAMWhDhrGOECgWZxoevMc5TlkObMqNSsVU62PYzXZ/SMTjzyGAFF1YusgxGcSWTEXBhp0CPwQ1A==
 
+cleaner-config@^0.1.3:
+  version "0.1.3"
+  resolved "https://registry.yarnpkg.com/cleaner-config/-/cleaner-config-0.1.3.tgz#99a4a7d421d0fa403be9890747780e6f10fba8c6"
+  integrity sha512-jKztmlT80wdSRNtY8vSfRLKRU90zsVPT0BKY4TaZ5zrtvDHpoB8F4Gg917OE3Xf5+f6mvxuJl0tgH2+13s0Hwg==
+  dependencies:
+    cleaners "^0.3.8"
+    minimist "^1.2.5"
+    sucrase "^3.17.1"
+
 cleaners@^0.3.2:
   version "0.3.2"
   resolved "https://registry.yarnpkg.com/cleaners/-/cleaners-0.3.2.tgz#e240a3837f5b2d59975f18d753dea7d1bab0fd4b"
   integrity sha512-SOjd7UOrU/8CF6V7QKZsfJB8MIBylrZOso+eJWYFaOm2jamKj7Wxbihg8vasnHzqtagv1vpwhN8xkD3liX/dIg==
 
+cleaners@^0.3.8:
```

Comment:
There are two versions of cleaners being installed in this repo. Not a big deal, but you can fix this by deleting these two sections of yarn.lock & reinstalling if you like.

Reply:
Hmm, this is because cleaner-config uses 0.3.8, and the sync server uses 0.3.2. Using `"resolutions"` in the package.json could give us a single version, but I'd rather just keep the two versions because that's the default behavior.

This did prompt me to upgrade cleaners for the sync server. So I added an upgrade commit.

Reply:
I take that back. No need for `"resolutions"`.